src/index.html

```diff
@@ -1,13 +1,47 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta charset="UTF-8">
     <title>Webpack Starter Kit</title>
   </head>
   <body>
-    <h1>Webpack Starter Kit</h1>
+    <nav>
+      <h1>HOTEL OVERLOOK</h1>
+      <h3 class="welcome-msg">WELCOME JULIA YOU AMAZING BITCH</h3>
+    </nav>
+    <main class="login-page hidden">
+      <section>
+        <div>
+          <label>USERNAME:</label>
```

Comment:
Remember that a `<label>` needs a `for` attribute that's identical to the `name` attribute of its matching `<input>` element. (I think I'm getting that right, but not 100% sure, so you should check.)